src/core/components/layout-utils/drop-down.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { PureComponent } from "react"
+import PropTypes from "prop-types"
+import cx from "classnames"
+
+export default class DropDown extends PureComponent {
+
+  static propTypes = {
+    id: PropTypes.string,
+    label: PropTypes.string,
+    displayLabel: PropTypes.string,
+    mod: PropTypes.string,
+    disbaled: PropTypes.bool,
+    children: PropTypes.node.isRequired
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      expanded: false
+    }
+  }
+
+  onClick = () => {
+    this.setState((state) => ({ expanded: !state.expanded }))
+  }
+
+  openDropdown = () => {
+    this.setState(() => ({ expanded: true }))
+    // set focus to first item/selected item
+  }
+
+  closeDropdown = () => {
+    this.setState(() => ({ expanded: false }))
+    // set focus to button
+  }
+
+  onKeyPress = (e) => {
```

Comment:
This could problematic cross browser - specifically IE11.
Will checkout what browsers they support